### DISCUSSION TITLE
Network commands

### DISCUSSION THREAD

**Discussion by mariusvasss** [RavenSystem/esp-homekit-devices#1517] [MEPLHAA Scripts]
I have two switches, one at the entrance to the house and one at the end of the hall. Please help me to make a JSON with network command, automations do not always work. I tried in every way but it didn't work out. I know the IP but I can't. Please help me.

**Comment by mariusvasss** [comment]
I tried the HTTP/TCP example config, doesn't work 😕

**Comment by RavenSystem** [comment]
First, post JSONs of both devices, and its IP address.

**Reply by mariusvasss** [reply]
{"c":{"l":13},"a":[{"0":{"h":[{"h":"192.168.100.67","u":"cm?cmnd=Power%20OFF"}],"r":[{"g":12}]},"1":{"h":[{"h":"192.168.100.67","u":"cm?cmnd=Power%20ON"}],"r":[{"g":12,"v":1}]},"b":[{"g":0,"t":1}]}]}

This is what i tried... Thank you for helping me!

**Reply by mariusvasss** [reply]
@RavenSystem can you help me pls? Thank you!
